### ISSUE TITLE
[Configuration] Disable "No Spam"

### ISSUE THREAD

**Issue by joemccann** [mehtadone/CryptoGramBot#66]
By default, if I have a lot of trades queued up I get:

> There are more than 30 poloniex trades to send. Not going to send them to avoid spamming you

I actually _want_ to receive them.  How can I disable this?  Is there a configuration option? @mehtadone

**Comment by rigide** [comment]
I agree. Why wouldn't you want to see them? It's in a separate chat window anyway..
